docs/en/source/opcua-IoTIntegration.rst

```diff
@@ -0,0 +1,627 @@
+.. figure:: figures/OPC-UA.png
+
+
+Integrate Scorpio Broker with OPC UA IoT agent 
+****************************************************
+
+This is a step-by-step tutorial that will introduce in detail how to enable OPC UA to FIWARE connecting an OPC UA server to Context Broker using the agent. The OPC UA data will be automatically published in a FIWARE Context Broker using NGSI data model.
+
+**What is OPC UA?**
+
+OPC UA is a well-known client-server protocol used in the Industry.
+
+In real environments, an OPC UA server is usually responsible for fetching sensor data from factory-level machinery making them available to an OPC UA client (the Agent in our case).
+
+Before a client can retrieve their values, sensors are mapped to the OPC UA Server Address Space as variables (or attributes). Moreover, it is also possible to control the machinery invoking methods exposed by the server.
+
+Sensor values access is provided through a subscription mechanism. For each sensor value the OPC UA client wants to have access to, it creates a subscription specifying some parameters. Using these parameters the client asks the server to send data according to some particular modalities. At that point the server determines if the requests can be fulfilled, otherwise it will continue sending data in a best effort mode.
+
+In our case the OPC UA Agent acts as bridge between the OPC UA server and the Scorpio Context Broker behaving as an OPC UA client.
+
+Actors
+****************************************************
+
+The actors involved in the scenario are:
+
+-   **OPC UA Server**, representing the data source
+-   **OPC UA Agent**, the connector to join industrial environment to FIWARE
+-   **Scorpio Context Broker**, the broker as entry point of FIWARE platform
```

Comment:
"Scorpio Broker" instead of "Scorpio Context Broker"

Reply:
Done